src/hooks/gatewayAuthToken/cache.ts

```diff
@@ -0,0 +1,27 @@
+import { LRUMap } from 'mnemonist';
+import { GatewayAuthTokenResponse } from 'types';
+
+// The limit of 10 is totally arbitrary. Felt like an okay number so all 3 tables could have
+//  something in cache and allow someone some flexibility to search
+const gatewayTokenCache = new LRUMap<string, GatewayAuthTokenResponse>(10);
```

Comment:
Moved the cache from local storage into LRU to make it a bit easier to work with. Also, we were always fetching new ones so it isn't like we were even using the local storage values after reloads anymore.